test/Microsoft.ML.Tests/TrainerEstimators/LbfgsTests.cs

```diff
@@ -80,14 +81,24 @@ public void TestLogisticRegressionStats_MKL()
                 s.StdComputer = new ComputeLRTrainingStdThroughHal();
             }));
 
-            var transformerChain = pipe.Fit(dataView) as TransformerChain<BinaryPredictionTransformer<ParameterMixingCalibratedPredictor>>;
+            var transformer = pipe.Fit(dataView) as TransformerChain<BinaryPredictionTransformer<ParameterMixingCalibratedPredictor>>;
 
-            var linearModel = transformerChain.LastTransformer.Model.SubPredictor as LinearBinaryModelParameters;
+            var linearModel = transformer.LastTransformer.Model.SubPredictor as LinearBinaryModelParameters;
             var stats = linearModel.Statistics;
             LinearModelStatistics.TryGetBiasStatistics(stats, 2, out float stdError, out float zScore, out float pValue);
 
             CompareNumbersWithTolerance(stdError, 0.250672936);
             CompareNumbersWithTolerance(zScore, 7.97852373);
+
+            var scoredData = transformer.Transform(dataView);
+
+            var coeffcients  = stats.GetCoefficientStatistics(linearModel, scoredData.Schema["Features"], 100);
```

Comment:
>GetCoefficientStatistics [](start = 37, length = 24)

I find it strange to have 
TryGetBiasStatistics which accept LinearModelStatistics object
and GetCoefficientStatistics which are part of  LinearModelStatistics object.
a) relationship to LinearModelStatistics object
b) why one is Try, and other is just Get?

Reply:
Yep, ugly. I am fixing all of it now, working on #2010

---
In reply to: [245155368](https://github.com/dotnet/machinelearning/pull/1979#discussion_r245155368) [](ancestors = 245155368)